microsoft-edge/webview2/release-notes/index.md

```diff
@@ -0,0 +1,1318 @@
+---
+title: Release Notes for the WebView2 SDK
+description: Release notes for Microsoft Edge WebView2, covering new features, APIs, and fixes for Win32, WPF, and WinForms.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 03/18/2024
+---
+# Release Notes for the WebView2 SDK
+
+These Release Notes provide information about new features and non-security updates that are included in the Microsoft Edge Stable Channel.
+<!-- todo: 
```

Comment:
Is the above sentence true for the WebView2 Prerelease SDK, or only for the WebView2 Release SDK?
@victorhuangwq

Reply:
This sentence should be for all of them:

It should be:
`These Release Notes provide information about new features and bug fixes that are included in the WebView2 Release SDK`

Reply:
"Release" is wrong, too narrow.  Prerelease as well.

Reply:
These Release Notes provide information about new features and bug fixes that are included in the WebView2 Release SDK and the WebView2 Prerelease SDK.

Reply:
Added coordinated lead-ins for the 3 articles